redfish-core/include/event_service_manager.hpp

```diff
@@ -372,52 +372,60 @@ class Subscription : public persistent_data::UserSubscription
 
     Subscription(const std::string& inHost, uint16_t inPort,
                  const std::string& inPath, const std::string& inUriProto) :
-        host(inHost),
-        port(inPort), policy(std::make_shared<crow::ConnectionPolicy>()),
-        client(policy), path(inPath), uriProto(inUriProto)
+        eventSeqNum(1),
+        host(inHost), port(inPort), path(inPath), uriProto(inUriProto)
     {
         // Subscription constructor
-        policy->invalidResp = retryRespHandler;
     }
 
     explicit Subscription(
         const std::shared_ptr<boost::asio::ip::tcp::socket>& adaptor) :
-        policy(std::make_shared<crow::ConnectionPolicy>()),
-        client(policy),
+        eventSeqNum(1),
```

Comment:
does this policy has any impact on 1050 HMC?

Reply:
I don't see any impact